source/cloud/cloud-administration/common-support-packet.rst

```diff
@@ -0,0 +1,22 @@
+Use the System Console to generate a Mattermost Support Packet that includes configuration information, logs, and data on external dependencies. Confidential data, such as passwords, are automatically stripped. 
+
+.. note:: 
+
+  System Admins should always sanitize the contents to remove sensitive information before sending the support packet to Mattermost by email or via a support ticket.
+
+To generate a support packet:
+
+1. Go to System Console, then select **Commercial Support** from the **Main Menu**.
+2. Select **Download Support Packet**. A zip file is downloaded to the local machine.
+
+A Mattermost Support Packet contains the following five files:
+
+- ``mattermost.log``
+- ``plugins.json``
+- ``sanitized_config.json``
+- ``support_packet.yaml``
+- ``warning.txt``
```

Comment:
`warning.txt` only appears if one of the files failed to be retrieved for whatever reason. It won't always be there if the above 4 files are retrieved without problem. Again not sure if it's important to point it out.